src/text-editor.coffee

```diff
@@ -609,15 +609,19 @@ class TextEditor extends Model
   # * "<filename>" when it is the only editing buffer with this file name.
   # * "<filename> — <unique-dir-prefix>" when other buffers have this file name.
   #
+  # * `textEditors` An optional {Array} of the other {TextEditor} instances that
```

Comment:
Ugh so many not great options 😬 

@as-cii and I talked about this in chat. Given that:

1. Doing It Right® would require some big deprecations, and
2. Doing It Wrong® doesn't leave us in a much better position than we're in today.

We're just gonna let this global dependency slide for now. Once we have a better idea of what/when 2.0, we can re-visit Doing It Right®.